apps/dashboard/app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Helpers for the projects page
+module ProjectsHelper
+  def render_readme(readme_location)
+    file_content = File.read(readme_location)
+
+    if File.extname(readme_location) == '.md'
+      markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+      markdown_html = markdown.render(file_content).html_safe
+      return sanitize(markdown_html)
+    elsif File.extname(readme_location) == '.txt'
+      # simple_format sanitizes its output
+      return simple_format(file_content)
```

Comment:
I wasn't going to mention this, but now that I see there's a conflict to resolve I may as well.

You don't need the `return` statements here. I use `solargraph` with vscode to see stuff like this. It would tell you you don't need them here. It's really helpful for small idioms like this.